packages/table/src/table.tsx

```diff
@@ -660,6 +711,22 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         });
     }
 
+    private maybeRenderCopyHotkey() {
+        const {getCellData} = this.props;
+        if (getCellData != null) {
+            return (
+                <Hotkey
+                    label="Copy selected table cells"
+                    group="table"
```

Comment:
Should group be capitalized? You can see the result my pressing `shift + ?` when the table is focused:
![image](https://cloud.githubusercontent.com/assets/569364/22606995/cb7ecaea-ea0a-11e6-8b91-6f48a29b3071.png)